athenacli/athenaclirc

```diff
@@ -14,6 +14,9 @@ role_arn = '' # e.g. arn:aws:iam::987654321000:role/MyAthenaAccessRole
 # The format is 's3://<your s3 directory path>'
 s3_staging_dir = ''
 
+# Athena workgroup used to run the query
+work_group = 'primary'
```

Comment:
I think this setting will override `AWS_ATHENA_WORK_GROUP` environment variable. Sometimes people don't want to use the `athenaclirc` and want to use environment variables. What happens if we pass '' or `None` to `pyathena`? will `pyathena` correctly use `primary` as default value?

Reply:
yes, when no workgroup is provided Athena defaults to `primary` workgroup. You may have noticed queries run with current version of `athenacli` are all executed in the `primary` workgroup

Reply:
cool, then could we set  `work_group` to `''` here? so users can use `AWS_ATHENA_WORK_GROUP` to override the default value.